### PR TITLE
[DPE-3644] Landscape client

### PR DESCRIPTION
**Pull request by dragomirp**
Adding basic tests for landscape-client subordinate.

### CODE REVIEW

File: tests/integration/test_subordinates.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+import pytest
+from pytest_operator.plugin import OpsTest
+
+from .helpers import (
+    CHARM_SERIES,
+    get_unit_address,
+)
+
+DATABASE_APP_NAME = "pg"
```
Comment (dragomirp) [inline tests/integration/test_subordinates.py:17]: The landscape bundle deploys a legacy postgresql

File: tests/integration/test_backups.py [hunk 2]
```diff
@@ -90,12 +90,6 @@ async def cloud_configs(ops_test: OpsTest, github_secrets) -> None:
             bucket_object.delete()
 
 
-@pytest.mark.group(1)
-async def test_none() -> None:
-    """Empty test so that the suite will not fail if all tests are skippedi."""
-    pass
-
-
```
Comment (dragomirp) [inline tests/integration/test_backups.py:98]: Skipped tests won't fail the job, so we don't really need this any more.

File: tests/integration/test_subordinates.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+LS_CLIENT = "landscape-client"
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.group(1)
+@pytest.mark.abort_on_fail
+async def test_deploy(ops_test: OpsTest, charm: str, github_secrets):
```
Comment (dragomirp) [inline tests/integration/test_subordinates.py:27]: When livepatch is added, tests should be skipped for forked repos.

**Review by taurus-forever — APPROVED** [review]
Nice! Tnx!

**Pull request merged**